hot/dev-server.js

```diff
@@ -4,9 +4,9 @@
 */
 /*globals window __webpack_hash__ */
 if(module.hot) {
-	var lastData;
+	var lastHash;
 	var upToDate = function upToDate() {
-		return lastData.indexOf(__webpack_hash__) >= 0;
+		return lastHash === __webpack_hash__;
```

Comment:
It's important to use `indexOf` because when using the multi-compiler the final hash is a concatentation of all child hashes.